readme.md

```diff
@@ -72,4 +72,10 @@ We take care of either lost responses or canceled futures by not holding handler
 All API methods related to the substrate node will be tested for operability and compatibility.
 Currently, we use [test containers](https://www.testcontainers.org/) and docker image [parity/substrate:v3.0.0](https://hub.docker.com/layers/parity/substrate/v3.0.0/images/sha256-1aef07509d757c584320773c476dcb6077578bbf2f5e468ceb413dcf908897f1?context=explore).
 
+## Building Locally
 
+You will need to install
```

Comment:
There is `asdf` support in the project and AFAIK this project takes a specific version of Rust to build (people have hit issues in the past). We can discuss this next week

Reply:
Changed the instructions to use `asdf` instead of `rustup` / `sdkman`